scripts/zkai_attention_kv_stwo_fine_grained_component_schema_gate.py

```
@@ -575,7 +575,7 @@ def validate_aggregate(aggregate: Any) -> None:
         raise StwoFineGrainedComponentSchemaGateError("json saving share drift")
     if aggregate["largest_component_saving_bucket"] != "fri_decommitment_merkle_path_bytes":
         raise StwoFineGrainedComponentSchemaGateError("largest component saving bucket drift")
-    if aggregate["largest_component_saving_bucket_bytes"] != 17_312:
+    if aggregate["largest_component_saving_bucket_bytes"] != 21_824:
         raise StwoFineGrainedComponentSchemaGateError("largest component saving bucket bytes drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)